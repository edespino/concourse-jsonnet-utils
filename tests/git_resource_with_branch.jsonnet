local concourse = import 'concourse.libsonnet';

{
  git: concourse.GitResource('git', 'https://github.com/sirech/example-concourse-pipeline.git', branch = 'devel'),
}
