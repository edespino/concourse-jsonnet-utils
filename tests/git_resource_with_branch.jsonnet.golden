{
   "git": {
      "name": "git",
      "source": {
         "branch": "devel",
         "uri": "https://github.com/sirech/example-concourse-pipeline.git"
      },
      "type": "git"
   }
}
